app_datetime/management/commands/convertdatetime.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+from app_datetime.models import DateTime
+from datetime import datetime
+import pytz
+from django.utils.translation import gettext as _
+
+
+class Command(BaseCommand):
+    help = 'Covert 10 PST Date/Time into UTC Date/Time'

```

Comment:
This command is related to converting `PST Date/Time` into `UTC Date/Time` and how many records it will update at a time will be provided as an option (with default value set to 10).

Reply:
As per requirement. The command should only convert 10 date/time that's why there is no option.